.circleci/src/jobs/build_android.yml

```diff
@@ -9,9 +9,17 @@ steps:
   - restore_ruby_cache:
       directory: android
   - restore_yarn_cache
+  - prepare_workspace
+  - restore_environment_variables
+  - run:
+      name: Base64 decode keystore env var
+      command: echo "${KEYSTORE_BASE64}" | base64 -di --decode > android/$KEYSTORE_FILENAME.gpg
+  - run:
+      name: Decrypt keystore
+      command: sudo gpg --passphrase ${KEYSTORE_ENCRYPTION_KEY} --pinentry-mode loopback -o "android/app/$KEYSTORE_FILENAME" -d "android/$KEYSTORE_FILENAME.gpg"
```

Comment:
Why did you decide to do this here instead of in fastlane? Also it kind of seems like this may print secrets in the CI :thinking:

Reply:
That's just how we do it in integreat as it is only used in fastlane anyway, so this should be fine as well.

Reply:
Will keep it this way, as fastlane is not needed, so I see only overhead and no benefit of doing it in fastlane.